packages/edit-site/src/components/page-templates-template-parts/add-new-template-part.js

```diff
@@ -58,3 +58,5 @@ export default function AddNewTemplatePart() {
 		</>
 	);
 }
+
+export default memo( AddNewTemplatePart );
```

Comment:
Was there anything in this component that made the re-renders expensive?

Reply:
I actually was testing "templates" mainly, so this one is probably less expensive but I just thought that it doesn't really make sense to re-render this one when you're interacting with the data views. (like filtering...)